### PR TITLE
[Tests, GPU, SLOW] fix a bunch of GPU hardcoded tests in Pytorch

### PR DESCRIPTION
**Pull request by patrickvonplaten**
in almost all tests I forgot to put the model on gpu via 
`model = model.to(torch_device)`

### CODE REVIEW

File: tests/test_modeling_ctrl.py [hunk 1]
```diff
@@ -219,6 +219,7 @@ class CTRLModelLanguageGenerationTest(unittest.TestCase):
     @slow
     def test_lm_generate_ctrl(self):
         model = CTRLLMHeadModel.from_pretrained("ctrl")
+        model = model.to(torch_device)
```
Comment (julien-c) [inline tests/test_modeling_ctrl.py:222]: models should just be model.to for consistency with other code

```suggestion
        model.to(torch_device)
```

Comment (julien-c) [reply tests/test_modeling_ctrl.py:222]: (and it saves a few chars)

Comment (patrickvonplaten) [reply tests/test_modeling_ctrl.py]: changed it everywhere :-)

**Review by julien-c — APPROVED** [review]

**Comment by julien-c** [comment]
Great!

**Review by sshleifer — APPROVED** [review]

**Pull request merged**